test/test_composite.jl

```diff
@@ -115,7 +117,7 @@ ftol = 1f-6
     u_scale ./= 2f0
     @test isapprox(u_scale, u)
     u_scale .= 2f0 .* u_scale .+ v_scale
-    @test isapprox(u_scale, 2f0 * u + 2f0 + v)
+    @test isapprox(u_scale, 2f0 * u + 2f0 + v,rtol=1f-7)
```

Comment:
Why the tolerance change, this was working before and should still do.

Reply:
Previously this w0 = judiWeights(rand(10, 10); nsrc=nsrc) is wrong and might lead to different results (since weights are in Float64). The weights are in default converted to Float32 so it does not pass the previous test. Now I slightly change it to `@test isapprox(u_scale, 2f0 * u + (2f0 + v))` which passes the test without changing the tolerance now.